### DISCUSSION TITLE
Close to tray

### DISCUSSION THREAD

**Discussion by Super90s** [OpenTabletDriver/OpenTabletDriver#3037] [Ideas]
It would be nice for the program to close to tray as a setting, because I find myself often accidentally closing the program instead of minimizing it to the tray, expecting it to close to tray.

**Comment by gonX** [comment]
Thanks for the idea, we're currently in the process of a huge UI overhaul, and this is actually the intention when tray functionality gets reimplemented in #2772. [Quote](https://discord.com/channels/615607687467761684/628651971267788800/1177227193038929921) from X9VoiD recently on Discord:

> for the tray functionality, let's follow the same conventions as other apps
>
> Minimize: we hide window but keep it visible in task bar
> Close: we hide window then hide it from task bar, cannot be alt-tabbed to

**Reply by Super90s** [reply]
Awesome! Glad to hear it!
